### PR TITLE
fix broken build and Jest incompatibility with ES6 imports

### PR DESCRIPTION
**Pull request by bp9320**
https://github.com/facebook/jest/issues/9395#issuecomment-583799300

### CODE REVIEW

File: src/context/calculation/CalculationState.test.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-let calculateCostBasis = require("./calculateCostBasis");
+// let calculateCostBasis = require("./calculateCostBasis");
```
Comment (zackjp) [inline src/context/calculation/CalculationState.test.js:1]: can remove commented out code

File: src/context/calculation/calculateCostBasis.js [hunk 2]
```diff
@@ -61,4 +61,5 @@ function calculateCostBasis(transactions) {
   return calculationExport;
 }
 
-module.exports = calculateCostBasis;
+// module.exports = calculateCostBasis;
```
Comment (zackjp) [inline src/context/calculation/calculateCostBasis.js:64]: can remove commented out code

File: src/context/calculation/CalculationState.test.js (hunk 1 above)
Comment (bp9320) [reply src/context/calculation/CalculationState.test.js]: done

File: src/context/calculation/calculateCostBasis.js (hunk 2 above)
Comment (bp9320) [reply src/context/calculation/calculateCostBasis.js]: done

**Pull request merged**